addons/web/static/src/legacy/js/fields/basic_fields.js

```diff
@@ -76,6 +76,32 @@ var TranslatableFieldMixin = {
 
 var DynamicPlaceholderFieldMixin = {
     DYNAMIC_PLACEHOLDER_TRIGGER_KEY: '#',
+    /**
+     * Overridable method that ensure the modelSelectorPopover is
+     * positioned properly.
+     *
+     * @public
+     * @param {ModelFieldSelectorPopover} modelSelector
+     */
+    positionModelSelector: function (modelSelector) {
+        const insertAfterElement = this.el;
```

Comment:
I don't see the added value of this variable. I'd just remove this variable and use `this.el` inline.